ui/src/datasource/DatasourceForm.vue

```diff
@@ -93,39 +134,57 @@
 import Vue from 'vue'
 import Component from 'vue-class-component'
 import { Emit, PropSync, Watch } from 'vue-property-decorator'
-
 import StepperButtonGroup from '../components/StepperButtonGroup.vue'
 import AdapterConfig from './edit/adapter/AdapterConfig.vue'
 import DatasourceMetadataConfig from './edit/DatasourceMetadataConfig.vue'
+import DatasourceSchemaEdit from './edit/schema/DatasourceSchemaEdit.vue'
 import TriggerConfig from './edit/TriggerConfig.vue'
 import Datasource from './datasource'
 import { requiredRule } from '../validators'
-
+import * as SchemaSuggestionREST from './schemaSuggestionRest'
 @Component({
-  components: { AdapterConfig, StepperButtonGroup, DatasourceMetadataConfig, TriggerConfig }
+  components: { AdapterConfig, StepperButtonGroup, DatasourceMetadataConfig, 
+    TriggerConfig, DatasourceSchemaEdit }
 })
 export default class DatasourceForm extends Vue {
+  
   private dialogStep = 1
-
   private validStep1 = false
   private validStep2 = true // starts with valid default values
   private validStep3 = true // starts with valid default values
   private validStep4 = true // starts with valid default values
+  private validStep5 = true // starts with valid default values
 
   @PropSync('value')
   private datasource: Datasource | undefined
 
+  private isSchemaAlive: boolean = false
   private required = requiredRule
 
   get isLoading (): boolean {
     return this.datasource === undefined
   }
 
+  get updateIsSchemaAlive (): boolean {
```

Comment:
Changes the state (`isSchemaAlive`), so should not be a getter. Please make it into a normal async private function that uses the `async/await` syntax. Use the Vue lifecycle hooks, like `mounted` to call the update method when the component gets rendered.